adapter.go

```diff
@@ -59,18 +60,30 @@ func finalizer(a *adapter) {
 
 // NewAdapter is the constructor for Adapter. If database name is not provided
 // in the Mongo URL, 'casbin' will be used as database name.
-func NewAdapter(url string, timeout ...interface{}) (persist.Adapter, error) {
-	if !strings.HasPrefix(url, "mongodb+srv://") && !strings.HasPrefix(url, "mongodb://") {
-		url = fmt.Sprint("mongodb://" + url)
+func NewAdapter(uri string, timeout ...interface{}) (persist.Adapter, error) {
```

Comment:
Why not url?

Reply:
Because I need to use package`net/url` If I use url as the variable name, it will conflict

Reply:
If you think the url must be used here, I canI can also give package an alias like this
```
import(
    neturl "net/url"
)
```

Reply:
The alias way sounds good